tests/sentry/test_wsgi.py

```diff
@@ -0,0 +1,32 @@
+import inspect
+import subprocess
+import sys
+
+
+def subprocess_test_wsgi_warmup():
+    # isort: off
+    import sentry.wsgi  # noqa
+    import sys
+
+    assert "django.urls.resolvers" in sys.modules
+
+    import django.urls.resolvers
+
+    resolver = django.urls.resolvers.get_resolver()
+    assert resolver._populated is True  # type: ignore[attr-defined]
+
+
+def test_wsgi_init():
+    """
+    This test ensures that the wsgi.py file correctly pre-loads the application and
+    various resources we want to be "warm"
+    """
+    subprocess_test = "\n".join(
+        line.lstrip() for line in inspect.getsource(subprocess_test_wsgi_warmup).splitlines()[1:]
+    )
```

Comment:
(irrelevant when you change to a string of code above -- but this should be `textwrap.dedent(source)` probably)